tests/test_serving.py

```diff
@@ -372,7 +372,7 @@ def app(environ, start_response):
 
 
 def test_port_must_be_integer():
-    def app(environ, start_response):
+    def app(_environ, start_response):
```

Comment:
`environ, start_response` is the standard WSGI callable signature. If we were to make this change, there are 27 occurrences of "parameter not used" in `tests/`.